### PR TITLE
Extracted a number of additional translations for TGMPA from GlotPress.

### PR DESCRIPTION
**Pull request by jrfnl**
Updated these with the most current tgmpa.pot file.
Purged old translations.
Made sure PO headers are correct.

Also:
Updated the existing translations to latest version of .pot & purged old text strings.

Pinging @elvishp2006 @hedii @vburlak @lewisporter :
All translation files have been updated against the latest .pot file in preparation for the release of TGMPA v 2.6.0. There have been a few string changes (not many).
If you have a chance, we'd very much appreciate it if you could do a quick review of your translation and update it if necessary based on the latest files as included in this PR.

### CODE REVIEW

**Comment by GaryJones** [comment]
For some reason, the "Required" and "Recommended" strings in en_GB have been given translations of "REQUIRED" and "RECOMMENDED". Please change them to lowercase and generate a new .mo file.

**Comment by hedii** [comment]
@jrfnl i added missing fields in #565 for french translation

**Comment by jrfnl** [comment]
Updated `en_GB` as per @GaryJones remark + merged @hedii PR (thanks!)

**Comment by vburlak** [comment]
Hi gents! Hi @jrfnl 😉  
I've updated russian translation files in #566. Cheers!

**Comment by vyskoczilova** [comment]
Hi, I have created a pull request with my Czech translation (my first time contributing, so hopefully it's done correctly) :) 

#570

File: languages/tgmpa-cs_CZ.po [hunk 1]
```diff
@@ -0,0 +1,499 @@
+# Translation of TGM-Plugin-Activation in Czech.
+# This file is distributed under the same license as the TGMPA package.
+msgid ""
+msgstr ""
+"Project-Id-Version: TGM Plugin Activation\n"
```
Comment (GaryJones) [inline languages/tgmpa-cs_CZ.po]: As per https://www.gnu.org/software/gettext/manual/html_node/Header-Entry.html, this field (for each file) should contain the project name AND version.

File: languages/tgmpa-de_DE.po [hunk 2]
```diff
@@ -1,63 +1,61 @@
+# Translation of TGM-Plugin-Activation in German (Germany)
+# This file is distributed under the same license as the TGMPA package.
 msgid ""
 msgstr ""
-"Project-Id-Version: TGM Plugin Activation\n"
-"POT-Creation-Date: 2016-01-21 17:25+0100\n"
-"PO-Revision-Date: 2016-01-22 13:49+0100\n"
+"Project-Id-Version: TGM-Plugin-Activation\n"
```
Comment (GaryJones) [inline languages/tgmpa-de_DE.po]: This value has hyphens within the project name; all values should be the same.

File: languages/tgmpa-en_GB.po [hunk 3]
```diff
@@ -0,0 +1,476 @@
+# Translation of TGM-Plugin-Activation in English (UK)
+# This file is distributed under the same license as the TGMPA package.
+msgid ""
+msgstr ""
+"Project-Id-Version: TGM-Plugin-Activation\n"
+"POT-Creation-Date: 2016-04-21 00:07+0200\n"
+"PO-Revision-Date: 2016-04-21 14:04+0200\n"
+"Last-Translator: \n"
```
Comment (GaryJones) [inline languages/tgmpa-en_GB.po]: You can put my name in here, as I was the one that translated the strings on GlotPress :-)

File: languages/tgmpa-de_DE.po (hunk 2 above)
Comment (jrfnl) [inline languages/tgmpa-de_DE.po]: So should there be hyphens in the project name or not ? Which is the correct way ?

Comment (GaryJones) [inline languages/tgmpa-de_DE.po]: No, there shouldn't.
"TGM Plugin Activation" is the project name.

File: languages/tgmpa-cs_CZ.po (hunk 1 above)
Comment (jrfnl) [inline languages/tgmpa-cs_CZ.po]: That is inconsistent with how GlotPress serves translations and fills in the `Project-Id-Version` header. Which should be leading in this case ?

Side-note - looks like I just discovered a bug in GlotPress - was checking how it served the headers and for themes it uses just the theme name (no version nr), for plugins it says 'stable (latest release)' without the plugin name - also for the header at the very top of the files.

Off to find out where I can fix/report this.

**Comment by jrfnl** [comment]
@GaryJones Updated the last commit in this PR for your remarks.

**Pull request merged**